tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation     Test 23-04 - VCH Inspect
+Resource          ../../resources/Util.robot
+Resource          ../../resources/Group23-VIC-Machine-Service-Util.robot
+Suite Setup       Setup
+Suite Teardown    Teardown
+
+Default Tags
+
+*** Keywords ***
+Setup
+    Start VIC Machine Server
+    Install VIC Appliance To Test Server  certs=${true}  additional-args=--ops-user=%{TEST_USERNAME} --ops-password=%{TEST_PASSWORD} --debug 1
+
+    ${id}=  Get VCH ID  %{VCH-NAME}
+    ${dc-id}=  Get Datacenter ID
+
+    Set Suite Variable  ${VCH-ID}  ${id}
+    Set Suite Variable  ${DC-ID}  ${dc-id}
+
+Teardown
+    Cleanup VIC Appliance On Test Server
+    Terminate All Processes    kill=True
+
+Inspect VCH
+    Get Path Under Target  vch/${VCH-ID}
+
+Inspect VCH Using Session
+    Get Path Under Target Using Session  vch/${VCH-ID}
+
+Inspect VCH Within Datacenter
+    Get Path Under Target    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Inspect VCH Within Datacenter Using Session
+    Get Path Under Target Using Session    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Verify VCH Inspect Output
+    # debug
+    Property Should Be Equal        .debug  1
```

Comment:
nit: this might be slightly more readable if you insert extra spaces between `.debug` and `1` so that `1` is vertically aligned with the other values, below

Reply:
ah, missed this one